llmfoundry/utils/builders.py

```diff
@@ -187,6 +188,44 @@ def build_icl_data_and_gauntlet(
     return icl_evaluators, logger_keys, eval_gauntlet_cb
 
 
+def build_load_planner(name: str) -> LoadPlanner:
+    """Builds a load planner from the registry.
+
+    Args:
+        name: Name of the load planner to build.
+
+    Returns:
+        LoadPlanner: The load planner.
+    """
+    return construct_from_registry(
+        name=name,
+        registry=registry.load_planners,
+        partial_function=True,
+        pre_validation_function=LoadPlanner,
+        post_validation_function=None,
+        kwargs={},
```

Comment:
are you sure you don't want the ability to pass in kwargs?

Reply:
Hm yeah it could be useful. How would you recommend taking kwargs from the yaml config?

```
fsdp_config:
    load_planner:
        custom_load_planner:
            arg1: ...
            arg2: ...
```
how does this look?

Reply:
added this